com.unity.render-pipelines.core/Editor/Lighting/ProbeVolume/ProbeGIBaking.cs

```diff
@@ -573,7 +576,7 @@ static void OnAdditionalProbesBakeCompleted()
                 {
                     // This scene has a reference volume authoring component in it?
                     ProbeVolumePerSceneData data = null;
-                    if (scene2Data.TryGetValue(scene, out data))
+                    if (scene2Data.TryGetValue(scene, out data) && data2Asset[data] != null)
```

Comment:
Picky comment, but I would replace that by `data2Asset.Contains(data)`, to remove the added if below, and above only add the element to the dictionnary if `hasAsset` is true

Reply:
I need to double check if somewhere we rely on having one entry in that array per scene; will check tomorrow, but if all is fine I do agree with you